WebAPI/appsettings.Development.json

```diff
@@ -0,0 +1,28 @@
+{
+  "Environment": "Development",
+  "CosmosDb": {
+    "Endpoint": "",
+    "Key": "",
+    "Database": "",
+    "Collection": ""
+  },
+  "ConnectionStrings": {
+    "DbModel": "data source=(LocalDb)\\MSSQLLocalDB;initial catalog=ProjectTira;integrated security=True;MultipleActiveResultSets=True;App=EntityFramework"
+  },
+  "Luis": {
+    "AppId": "3eda5e77-1471-41f5-ae17-a7f5e0825b30",
+    "SubscriptionKey": "f26d57150270448ca1986bb7e4a08ad3",
+    "EndpointUrl": "https://westus.api.cognitive.microsoft.com"
+  },
+  "Maps": {
+    "SubscriptionKey": "ay41VKwaVczc7rlvS9krCupc_OQybqGBLGFz9IsDZoc",
+    "SearchUrlFormat": "https://atlas.microsoft.com/search/fuzzy/json?api-version=1.0&subscription-key={0}&format=json&query={1}"
```

Comment:
Make sure not to check in the api keys (I've done it on accident before) - you can save a copy locally so that you can connect for development, but eventually these will just be stored in the azure instances

Reply:
You can also remove the ones that this app doesn't need, like luis, maps, cosmos - same with the other appsettings file